docs/Configuration-Settings.md

```diff
@@ -180,11 +180,26 @@ number.
     specified for `tachyon.master.hostname`.</td>
 </tr>
 <tr>
+  <td>tachyon.master.bind.host</td>
+  <td>0.0.0.0</td>
+  <td>The hostname Tachyon's master node binds to.</td>
```

Comment:
Is there any meaningful use for this value of `tachyon.master.bind.host` to be neither the value of`tachyon.master.hostname` nor `0.0.0.0`?
If that is not allowed, we shall be more explicit here about the possible values.